include/TrustWalletCore/TWEthereumChainID.h

```diff
@@ -10,6 +10,7 @@
 
 TW_EXTERN_C_BEGIN
 
+/// [DEPRECATED] Deprecated after 2.9.2, use CoinType.ChainId to obtain chainID of a coin.
```

Comment:
Let's codegen this file like `TWHRP.h`, filter by blockchain type etc

Reply:
It's possible to generate, I'm not sure if it is of much use though.

Reply:
App registers them as default chainId values (access the enum directly), now have to call into wallet core multiple times if using CoinType.chainId